risc0/zkvm/src/guest/mod.rs

```diff
@@ -101,28 +99,17 @@ core::arch::global_asm!(include_str!("memcpy.s"));
 #[cfg(target_os = "zkvm")]
 mod libm_extern;
 
-fn _fault() -> ! {
-    #[cfg(target_os = "zkvm")]
-    unsafe {
-        asm!("sw x0, 1(x0)")
-    };
-    unreachable!();
-}
-
 /// Aborts the guest with the given message.
 pub fn abort(msg: &str) -> ! {
     // A compliant host should fault when it receives this syscall.
```

Comment:
Perhaps replace this comment (that is now not entirely accurate since it doesn't matter if the host is compliant) with a `SAFETY:`